cppcache/src/SerializationRegistry.cpp

```diff
@@ -287,25 +285,21 @@ void SerializationRegistry::addPdxType(TypeFactoryMethodPdx func) {
   theTypeMap.bindPdxType(func);
 }
 
-void SerializationRegistry::addType(int64_t compId, TypeFactoryMethod func) {
-  theTypeMap.rebind(compId, func);
+void SerializationRegistry::addType(int32_t id, TypeFactoryMethod func) {
+  theTypeMap.rebind(id, func);
 }
 
-void SerializationRegistry::removeType(int64_t compId) {
-  theTypeMap.unbind(compId);
-}
+void SerializationRegistry::removeType(int32_t id) { theTypeMap.unbind(id); }
 
 void SerializationRegistry::addType2(TypeFactoryMethod func) {
```

Comment:
Should be `addDataSerializableFixedId(...)` and `rebindDataSerializableFixedId`.